stock_analytic/models/stock_move.py

```diff
@@ -72,6 +72,7 @@ def _action_done(self, cancel_backorder=False):
                     "company_id": move.company_id.id,
                 }
             )
+            move.move_line_ids.analytic_distribution = move.analytic_distribution
```

Comment:
`move.move_line_ids.analytic_distribution = move.analytic_distribution` should be on line 62. 
Otherwise, during moves such as receipts, the analytic distribution of move_lines will not be updated.